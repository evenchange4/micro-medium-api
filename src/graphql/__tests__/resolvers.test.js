const resolvers = require('../resolvers');

jest.mock('../../utils/parser', () => ({
  posts: data => data,
  user: data => data,
}));

it('should resolve query posts', async () => {
  expect(
    await resolvers.Query.posts(
      {},
      {
        username: 'evenchange4',
        limit: 10,
      },
      {
        Raw: {
          load: data => new Promise(resolve => resolve(data)),
        },
      },
    ),
  ).toMatchSnapshot();
});

it('should resolve query user', async () => {
  expect(
    await resolvers.Query.user(
      {},
      {
        username: 'evenchange4',
      },
      {
        Raw: {
          load: data => new Promise(resolve => resolve(data)),
        },
      },
    ),
  ).toMatchSnapshot();
});
