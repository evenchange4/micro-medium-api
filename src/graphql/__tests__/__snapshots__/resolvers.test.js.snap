// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should resolve query posts 1`] = `
Object {
  "limit": 10,
  "username": "evenchange4",
}
`;

exports[`should resolve query user 1`] = `
Object {
  "username": "evenchange4",
}
`;
